.gitignore

```
@@ -1,2 +1,3 @@
 /target
 /result
+.env
```

Every file needs to end with a line break.
```suggestion
.env

```